lib/internal/test_runner/utils.js

```diff
@@ -200,6 +200,7 @@ function parseCommandLine() {
   let functionCoverage;
   let destinations;
   let isolation;
+  let testCwd = getOptionValue('--experimental-test-cwd') || process.cwd()
```

Comment:
Could the `cwd` be set on the C++ side as the default?

Reply:
I've answered this in the other comments as well, but yes, I would definitely prefer to have a default set on the C++ side